src/aks-preview/azext_aks_preview/agentpool_decorator.py

```diff
@@ -436,6 +436,68 @@ def get_skip_gpu_driver_install(self) -> bool:
 
         return skip_gpu_driver_install
 
+    def get_enable_secure_boot(self, enable_validation: bool = False) -> bool:
```

Comment:
Since you have given the verification responsibility only to enable_xxx, I think you can remove parameter `enable_validation`. In any case, we should verify that there is no conflict between the two, right?